src/turbomind/triton_backend/CMakeLists.txt

```diff
@@ -34,250 +34,3 @@ set_property(TARGET TransformerTritonBackend PROPERTY POSITION_INDEPENDENT_CODE
 install(TARGETS TransformerTritonBackend DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 add_subdirectory(llama)
-
-# Needn't build triton backend on windows
-if (MSVC)
-  return ()
-endif()
-
-#
-# Options
-#
-option(TRITON_ENABLE_GPU "Enable GPU support in backend" ON)
-option(TRITON_ENABLE_STATS "Include statistics collections in backend" ON)
-
-set(TRITON_PYTORCH_INCLUDE_PATHS "" CACHE PATH "Paths to Torch includes")
-set(TRITON_PYTORCH_LIB_PATHS "" CACHE PATH "Paths to Torch libraries")
-
-set(TRITON_BACKEND_REPO_TAG "r22.12" CACHE STRING "Tag for triton-inference-server/backend repo")
```

Comment:
Since we have removed the dependency on triton here, there is no longer any concern about the version or tag of triton. In this case, do we still need to create a new Dockerfile on top of r24.03's docker? ref https://github.com/InternLM/lmdeploy/pull/1986#issuecomment-2231302211